src/components/Job/JobTable.vue

```diff
@@ -65,38 +59,37 @@ export default {
 
 <template>
   <v-data-table
-    :headers="headers_"
+    :headers="headers"
     :items="items"
     :server-items-length="serverItemsLength"
     :options="options"
     item-key="_id"
     @update:options="$emit('update:options', $event)"
   >
     <template #item="props">
-      <tr @click="$emit('job-click', $event, props.item)">
-        <td class="one-line">
-          {{ props.item.title }}
-        </td>
-        <td class="one-line">
-          {{ props.item.type }}
-        </td>
-        <td class="one-line">
-          {{ props.item.updateString }}
-        </td>
-        <td
-          :title="props.item.statusText"
-          class="one-line"
-          nowrap="nowrap"
-        >
-          <job-progress :formatted-job="props.item" />
-        </td>
-        <td class="one-line pa-0">
-          <slot
-            v-bind="props"
-            name="jobwidget"
-          />
-        </td>
-      </tr>
+      <slot
+        v-bind="props"
+        name="jobrow"
+      >
```

Comment:
I'm thinking this and the headers prop may be overkill, and if you want this table to look different, just write your own from scratch.

Reply:
Do we have use cases in the wild for extending this component? I'd avoid speculating about it until we have at least one.